cmd/podman/containers/stop.go

```diff
@@ -124,6 +128,10 @@ func stop(cmd *cobra.Command, args []string) error {
 		stopOptions.Filters[split[0]] = append(stopOptions.Filters[split[0]], split[1])
 	}
 
+	// If no args, then nothing to stop
```

Comment:
This looks unrelated to the cidfile bug, is it?

Reply:
Well if you do a --cidfile=/tmp/bogus --ignore, then args will be [].  Currently this works correctly, but I do worry about calling a function that will return right  a way, in case someone down the line changes the code to report an error when args=[].

Reply:
Thanks!  I should have expanded the entire function.